scripts/download_file.py

```diff
@@ -227,15 +237,15 @@ def compare_md5(remote_md5: str, local_md5: str) -> str:
             for fname in temp_files:
                 with open(fname, "rb") as fh:
                     chunk = fh.read(BLOCK_SIZE)
-                    while chunk:
+                    while chunk is not None:
```

Comment:
I did not test, but likely the same issue as above.

Reply:
Addressed.